src/main/resources/add-ons-to-index.json

```diff
@@ -43,6 +43,27 @@
       }
     },
     {
+    "uid": "org.openmrs.owa.addonmanager",
+    "type": "OWA",
+    "name": "Add-On-Manager",
+    "maintainers": [
+      {
+        "name": "Andela apprentices",
+        "url": "https://andela.com/"
+      },
+      {
+        "name": "OpenMRS",
+        "url": "https://github.com/openmrs"
+      }
+    ],
+    "backend": "org.openmrs.addonindex.backend.Bintray",
+    "bintrayPackageDetails": {
+      "owner": "openmrs",
+      "repo": "owa",
+      "package": "Add-On-Manager"
```

Comment:
Can you also change the package name to be in the same format as that for other OWAs?

Reply:
I've made the change